src/raftstore/store/msg.rs

```diff
@@ -124,6 +125,32 @@ pub enum Tick {
     CleanupImportSST,
 }
 
+// Hardcode the length for now, maybe can use macro to generate it later.
+const TICK_NAMES: [&str; 12] = [
```

Comment:
I think we can just format the `Tick` in debug format in the `timeout` function, it should not be a performance problem.
If it really hurt performance, please use a more decent way to generate it.